CONTRIBUTING.md

```diff
@@ -53,7 +59,7 @@ To add a dependency via Yarn, for example, run `yarn add <dependency>` from with
 
 ## Installing Go
 
-Chronograf requires Go 1.10 or higher.
+Chronograf requires Go 1.13 or higher.
```

Comment:
If chronograf still builds with 1.10/1,11, technically it's still supported, but this is fine.